.devcontainer/README.md

```diff
@@ -2,6 +2,40 @@
 
 For a comprehensive developer environnment supporting AGW c / c++ workflows (extensible with ~effort to other Magma uses) - see the instructions below.
 
+## Prerequisites and Restrictions
+
+DevContainer is a technology developed by Microsoft. Thus, it is only available for [_VS Code_](https://code.visualstudio.com/docs/remote/containers), but **not** for _IntelliJ_. However, as Github is also currently owned by Microsoft, Github provides a DevContainer integration named ["Codespaces"](https://docs.github.com/en/codespaces).
+In order to use the Github codespaces, you must be member of the Magma Github organisation.
```

Comment:
There are a couple `misspell` GH Action findings in this PR, would you mind addressing? I don't think this is a colour vs color type of thing but I could be wrong =D

Reply:
Yes, it seems that `misspell` favours the American English over British English, but at my school we were rather trained in BE flavour. :innocent: I will go over it again, if the `misspell` complains.